src/browser/topbar.ts

```diff
@@ -68,11 +68,12 @@ const repaint = () => {
   ctx.stroke()
 }
 const createCanvas = () => {
-  canvas = document.createElement('canvas')
+  const canvas = document.createElement('canvas')
+  canvas.ariaHidden = 'true'
   const style = canvas.style
   style.position = 'fixed'
-  style.top = style.left = style.right = style.margin = style.padding = 0
-  style.zIndex = 100001
+  style.top = style.left = style.right = style.margin = style.padding = '0'
```

Comment:
Is it intentional? Is a string a better/proper way to code it? If yes - I'm 👍 

Same q for the next line

Reply:
> Is it intentional? Is a string a better/proper way to code it? If yes - I'm 👍

Yes, it is proper way according to `HTMLCanvasElement` type 😊 When I added `const` locally, the linter instantly became unhappy
![Screenshot 2024-11-14 at 15 09 55](https://github.com/user-attachments/assets/a72fa180-545b-427c-88cb-f77c51b664e6)